go/rep_addr_info.go

```diff
@@ -0,0 +1,101 @@
+package pb
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/multiformats/go-multiaddr"
+)
+
+// HostInfo is an interface that we can build a AddrInfo from.
```

Comment:
Small typo, should be `that we can build an AddrInfo from.`

Reply:
thanks